hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/oauth/OAuthHandler.java

```diff
@@ -0,0 +1,14 @@
+package pl.allegro.tech.hermes.consumers.consumer.oauth;
+
+import pl.allegro.tech.hermes.api.SubscriptionName;
+import pl.allegro.tech.hermes.consumers.consumer.sender.MessageSendingResult;
+
+public interface OAuthHandler {
```

Comment:
it's not that its only oAuth - i would really prefer to see some more generic names here and there, especially in the interfaces